modules/pytket-qiskit/docs/intro.txt

```diff
@@ -13,8 +13,8 @@ representations, simulation and access to the `IBMQ <https://www.research.ibm.co
 run on IBM backends and simulators, as well as conversion to and from Qiskit
 representations.
 
-``pytket-qiskit`` is available for Python 3.7, 3.8 and 3.9, on Linux, MacOS and Windows. To
-install, run:
+``pytket-qiskit`` is available for Python 3.8, 3.9 and 3.10, on Linux, MacOS and
```

Comment:
Is there a known problem with M1 macs? Should we bring this up here? Or is that solved?

Reply:
There is, but it is a qiskit issue. We don't mention M1 Mac support here, though it would probably be worth having a page somewhere explaining the known issues.

Reply:
I think it would be helpful to document that somewhere. Not sure what the best place is to do that. Maybe somewhere central in the pytket docs for all known problems?